### PR TITLE
Adding a support submission button on CLI to assist in support

### PR DESCRIPTION
**Pull request by blckmn**
There are a couple of things I would like to eventually tidy up here.

1. There needs to be a better way to identify when the "output" is completed, following the issuing of a command.
2. The presets use a cliEngine, yet the cli doesn't. I feel there is a lot of duplication here, and this can be simplified.

Both of the above are not for this PR to solve, but I will look into them into the future.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by chmelevskij — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
